packages/private/edtr-io/src/plugins/solution-steps/index.ts

```diff
@@ -19,12 +20,33 @@ const stepState = object({
   content: child({ plugin: 'rows' })
 })
 
-export const solutionStepsState = object({
+const solutionStepsState_v0 = object({
   introduction: child({ plugin: 'text' }),
   strategy: optional(child({ plugin: 'rows' })),
   solutionSteps: list(stepState),
   additionals: optional(child({ plugin: 'rows' }))
 })
+const solutionStepsState_v1 = object({
+  introduction: child({ plugin: 'rows' }),
+  strategy: optional(child({ plugin: 'rows' })),
+  solutionSteps: list(stepState),
+  additionals: optional(child({ plugin: 'rows' }))
+})
+
+export const solutionStepsState = migratable(solutionStepsState_v0).migrate(
```

Comment:
why do you have a migratable if you migrate the state in `deserialize` by hand?

Reply:
remove migrate